.editorconfig

```diff
@@ -1,11 +1,14 @@
+# https://editorconfig.org
 root = true
 
 [*]
+charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = space
 tab_width = 4
+max_line_length = 100
```

Comment:
Why 100? By PSR-2 (at least for PHP) it said:
> There MUST NOT be a hard limit on line length; the soft limit MUST be 120 characters; lines SHOULD be 80 characters or less.
So, 120 fits more.

Reply:
It's a soft limit, I don't think we should enforce a max line length.

Reply:
I agree @JoshuaLuckers Probably it should be removed or set as `off` to allow use editor settings.

Reply:
I set the value `max_line_length` to `off`